src/nft/components/details/detailsV2/TabbedComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { containerStyles, containerXPadding } from './shared'
+
+const TabbedContentContainer = styled(Column)<{ noContentPadding?: boolean }>`
+  width: 100%;
+  align-self: flex-start;
+
+  ${containerStyles}
+  ${({ noContentPadding }) => !noContentPadding && containerXPadding}
+`
+
+const TabsRow = styled(Row)<{ selfPadding?: boolean }>`
+  gap: 32px;
+  margin-bottom: 12px;
+  width: 100;
+
+  ${({ selfPadding }) => selfPadding && containerXPadding}
+`
+
+const Tab = styled(ThemedText.SubHeader)<{ isActive: boolean }>`
+  color: ${({ theme, isActive }) => (isActive ? theme.textPrimary : theme.textTertiary)};
+  line-height: 24px;
+  cursor: pointer;
+
+  &:hover {
+    opacity: ${({ theme }) => theme.opacity.hover};
+  }
+`
+
+export interface Tab {
+  title: React.ReactNode
+  key: string
+  content: JSX.Element
+}
+
+interface TabbedComponentProps {
+  tabs: Tab[]
+  defaultTab: Tab
+  noContentPadding?: boolean
```

Comment:
I don't think it's worth having such a specific parameter be required across all the components since we see such variation, I think it might be better to handle it at atomic container level